runtime/arm64.S

```diff
@@ -1029,6 +1055,7 @@ L(do_perform):
         ldr     x9, Caml_state(current_stack)
         SWITCH_OCAML_STACKS x9, x10
     /*  No parent stack. Raise Effect.Unhandled. */
+        ENTER_FUNCTION
```

Comment:
This looks incorrect to me - after the TSAN block, this ends with
`b G(caml_c_call)`, which does the `ENTER_FUNCTION`.

amd64 needs `ENTER_FUNCTION` here because it jumps not in the beginning of `caml_c_call` but a bit later down, skipping the `ENTER_FUNCTION`, and compensates here.

Reply:
Understood, I've removed that part as it was clearly wrong.